ror-senior-dev/README.md

```diff
@@ -69,7 +111,43 @@ have feature. So it is a bonus task & you can finish it if you have time.
 
 #### (3) TST-0003 - *Acceptance criteria*: Return the fastest sailing legs (direct or indirect) in the same above format
 
-As those are small changes, we can create one branch for all the changes.
+#### Input
+```json
+CNSHA
+NLRTM
+fastest
+```
+
+#### Output
+```json
+[
+  {
+    "origin_port": "CNSHA",
+    "destination_port": "ESBCN",
+    "departure_date": "2022-01-29",
+    "arrival_date": "2022-02-06",
+    "sailing_code": "ERXQ",
+    "rate": "261.96",
+    "rate_currency": "EUR"
+  },
+  {
+    "origin_port": "ESBCN",
+    "destination_port": "NLRTM",
+    "departure_date": "2022-02-16",
+    "arrival_date": "2022-02-20",
+    "sailing_code": "ETRG",
+    "rate": "69.96",
+    "rate_currency": "USD"
+  }
+]
+```
+
+### Project Requirements
+
+1. Please, create one single branch for all the changes.
+2. Make sure your app run on docker and all the dependencies are included on it
+3. Create a private project and invite "joassouza", "yamanaltereh" once you're done
```

Comment:
@joassouza I do not like that 😂 
what is the point here? why not just keep uploaded a zip file with git history

Reply:
the idea is to avoid lots of public repo where people can just copy and paste the solution

Reply:
what do you suggest?

Reply:
no definitely not public, why not just send zip file including .git like usual?

Reply:
usually they don't send the zip file. I upload the zip file on recruitee